readme.md

```diff
@@ -216,7 +216,10 @@ If you're a Python or Django developer, fork the repo and get stuck in! If you'd
 
 If you change content or images in this repo and need to prepare a new fixture file for export, do the following on a branch:
 
-`./manage.py dumpdata --natural-foreign --indent 2 -e auth.permission -e contenttypes -e wagtailcore.GroupCollectionPermission -e wagtailimages.filter -e wagtailcore.pagerevision -e wagtailimages.rendition -e sessions > bakerydemo/base/fixtures/bakerydemo.json`
+```bash
+./manage.py dumpdata --natural-foreign --indent 2 -e auth.permission -e contenttypes -e wagtailcore.GroupCollectionPermission -e wagtailcore.revision -e wagtailimages.rendition -e sessions -e wagtailsearch.indexentry -e wagtailsearch.sqliteftsindexentry -e wagtailcore.referenceindex -e wagtailcore.pagesubscription -e wagtailcore.pagelogentry > bakerydemo/base/fixtures/bakerydemo.json
```

Comment:
Should we add `wagtailcore.modellogentry` as well?

Reply:
Good point!